modules/social_features/social_search/src/EventSubscriber/RedirectSubscriber.php

```diff
@@ -102,8 +102,8 @@ public function redirectSearchWithPrefilledExposedFilters(RequestEvent $event) {
     $query = $this->requestStack->getCurrentRequest()->query->all();
 
     // Workaround for drupal.org issue #3085806.
-    if (empty($query) || empty($query['created_op'])) {
-      $query = ['created_op' => '<'];
+    if (is_array($query) && (empty($query) || empty($query['created_op']))) {
```

Comment:
`(empty($query) || empty($query['created_op'])`

These checks are quite ambiguous. $query is empty then the array merge doesn't make sense below.